sdks/go/container/build.gradle

```diff
@@ -38,8 +41,10 @@ docker {
   files "./build/"
   buildArgs(['pull_licenses': project.rootProject.hasProperty(["docker-pull-licenses"]) ||
                      project.rootProject.hasProperty(["isRelease"])])
-  buildx project.containerPlatforms() != [project.nativeArchitecture()]
+  buildx useBuildx
   platform(*project.containerPlatforms())
+  load useBuildx && !pushContainers
```

Comment:
I am not sure how useful it is to keep the image designed for a different platform. we should test and see that all layers (incl base) would be selected for the target platform after such loading + manual push to a different repo.